### PR TITLE
Edited comments

### PR DESCRIPTION
**Pull request by FosterMichelle**
@rwike77 This pull request supersedes #20

### CODE REVIEW

File: src/Microsoft.ServiceFabric.Actors/Runtime/StateChangeKind.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@ namespace Microsoft.ServiceFabric.Actors.Runtime
     using System.Threading;
 
     /// <summary>
-    /// Represents kind of state change for an actor state when 
+    /// Represents a kind of state change for an actor state when
```
Comment (rwike77) [inline src/Microsoft.ServiceFabric.Actors/Runtime/StateChangeKind.cs]: Should be "Represents the kind of state change..."

**Comment by rwike77** [comment]
@amanbha  These updates look OK to me.

**Comment by FosterMichelle** [comment]
@rwike77 edited StateChangeKind.cs as requested

**Comment by rwike77** [comment]
@FosterMichelle  Thanks.  @amanbha Looks good to me now.

**Pull request merged**